kin-sdk-core/src/main/java/kin/sdk/core/EncryptorImplV16.java

```diff
@@ -0,0 +1,18 @@
+package kin.sdk.core;
+
+
+/**
+ * UNSAFE no-op encryptor for below api 18, it will do no encryption
+ */
+class EncryptorImplV16 implements Encryptor {
```

Comment:
add disclaimer on readme for this

Reply:
I will took it as a separate discussion how to handle version 16